Examples/Assets.xcassets/nps-picnic-area.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "32px-Pictograms-nps-picnic_area.svg.png",
```

Comment:
Since we’re evidently sourcing these from SVG, let’s use PDF here instead of a single PNG — that way we only have to provide one file to cover all scales.

Reply:
It may also make sense to put these in their own `xcassets` catalog, so that we can have developers download that directly.